Testing/init_dev_certs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
```

Comment:
Another option is to use the go tools. Example:

go run $(go env GOROOT)/src/crypto/tls/generate_cert.go --host=localhost:9000,127.0.0.1:9000 --ecdsa-curve=P256 --ca=true

Reply:
For it to be a valid codesigning cert, you also specifically need the extended properties included in `openssl.cnf`.